configure.ac

```diff
@@ -434,6 +434,16 @@ AS_CASE([$FIPS_VERSION],
     ]
 )
 
+# Use FIPS with external crypto, wolfTPM with a FIPS approved TPM for example
+AC_ARG_ENABLE([hybrid-fips-tpm],
```

Comment:
I don't think you need a new build option or configure for this. If you have FIPS enabled and have crypto callback both enabled (with devId set) then it should use this behavior by default.